src/views/LoanDetails/LoanDetails.js

```diff
@@ -57,7 +57,7 @@ import loanActionMap from '../../components/data/static/loanActionMap';
 import LoanProxyDetails from './LoanProxyDetails';
 
 import css from './LoanDetails.css';
-import { localizeCurrencyAmount } from '../../components/util/localizeCurrencyAmount';
+import localizeCurrencyAmount from '../../components/util/localizeCurrencyAmount';
```

Comment:
after updating` src/hooks/index.js` file on the lines of other hooks exports (as suggested below in one of the comments), adjust this import